virttest/env_process.py

```diff
@@ -526,7 +527,11 @@ def validate_memory_resource():
         if (params.get("setup_ksm") == "yes" and
                 params.get("ksm_run", "1") == "1"):
             magnification = 1.2
-        free_mem = "%s KB" % memory.freememtotal()
+        # Test whether is MemAvailable already introduced
+        if 'MemAvailable' in genio.read_file('/proc/meminfo'):
+            free_mem = "%s KB" % memory.read_from_meminfo('MemAvailable')
```

Comment:
From my point of view, I'm afraid not fully endorses the view of this article about the exception usage in Python, a typical usage is catching ImportError during import modules. But, of course I'm not insist on this. 
BTW, let's see if anyone else has comment on this, they may have more experienced. @lmr @ldoktor